### PR TITLE
Add documentation and logo for Aladdin Connect cover

### PR DESCRIPTION
**Pull request by shoejosh**
**Description:**
Documentation for Aladdin Connect Cover

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15699

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thank you, @shoejosh! Those are some pretty nice additions to the project 👍 
✅ Approved. Can be merged as soon as the parent PR get merged.

File: source/_components/cover.aladdin_connect.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+description: "Instructions how to integrate Genie Aladdin Connect garage door covers into Home Assistant."
+date: 2018-07-26 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: aladdin_connect.png
+ha_category: Cover
+ha_release: 0.75
+---
```
Comment (MartinHjelmare) [inline source/_components/cover.aladdin_connect.markdown]: Please add `ha_iot_class` to the front matter. I assume the value should be `Cloud Polling`.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/cover.aladdin_connect.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/cover.aladdin_connect.markdown]: Well spotted @MartinHjelmare!

**Comment by frenck** [comment]
@shoejosh, can you take a look? 👍

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**